hooks/useAuthStore.ts

```diff
@@ -47,24 +49,28 @@ export const useAuthStore = create<AuthState>((set) => ({
   signUp: async (email: string, password: string) => {
     try {
       set({ isLoading: true, error: null });
-      const { data, error } = await supabase.auth.signUp({ email, password });
-      if (error) throw error;
-      set({ session: data.session, isLoading: false });
+      const { error } = await supabase.auth.signUp({ email, password });
+      if (error) {
+        throw new Error(error.message);
+      }
+      set({ isLoading: false, error: null });
+      return true;
```

Comment:
We don't need to return anything from these functions (signIn as well)

Reply:
Otherwise this looks good!

Reply:
Understood, this was me trusting AI suggestions without due skepticism. I'll update to infer success based on absence of errors

Reply:
I'm a little confused though because now you have 2 PRs out that make changes in this file and this PR was supposed to be about creating a pressable component ?

I think the best thing to do is to copy the changes you've made on this branch, remove them and commit, push. 

Then go back to where you started changing this logic on https://github.com/Wheely-Good/wheels-expo/pull/11 and update there and also reflect the changes in the title / description.

Reply:
Yeah, I started this branch from main but I wanted to see the updates to the sign-up from my other branch. I followed instructions using git cherry-pick as suggested by Claude but I think something went wrong along the way. Is there a different git command I can't look into that will allow me to bring some changes from one feature branch into a different feature branch so I can refer to them, but then not include those updates when I commit the second feature branch?